src/Motorcycle/Sport.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Motorcycle;
+
+class Sport implements MotorcycleInterface
```

Comment:
I didn't understand that Sport is a bike, i thought it is about some sports like football. Maybe you can rename it to SportBike or something similiar